webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -1,23 +1,24 @@
 <template>
-  <ds-button v-if="totalNotifications <= 0" class="notifications-menu" disabled icon="bell">
-    {{ totalNotifications }}
+  <ds-button v-if="!notificationsCount" class="notifications-menu" disabled icon="bell">
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-20T21:00:44Z" title="Friday, September 20th 2019, 11:00:44 pm +02:00">Sep 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -18,6 +18,7 @@
 
 <script>
 import Dropdown from '~/components/Dropdown'
+import { REFRECH_MILISEC } from '~/constants/notifications'
```


```suggestion
import { REFRESH_MILLISEC } from '~/constants/notifications'
```